systemtest/playlist.xml

```diff
@@ -533,6 +553,7 @@
 	perl $(TEST_RESROOT)$(D)stf$(D)stf.core$(D)scripts$(D)stf.pl \
 	-test-root=$(Q)$(TEST_RESROOT)$(D)stf;$(TEST_RESROOT)$(D)openjdk-systemtest;$(TEST_RESROOT)$(D)openj9-systemtest$(Q) \
 	-systemtest-prereqs=$(Q)$(TEST_RESROOT)$(D)systemtest_prereqs$(Q) \
+	-java-args=$(Q)$(JVM_OPTIONS)$(Q) \
```

Comment:
we already have a `-java-args=$(Q)-Xdisableexcessivegc$(Q) \`, I think the `-java-args=$(Q)$(JVM_OPTIONS)$(Q) \` can be combined with it.

Reply:
Good catch! Thanks.

Reply:
just to confirm, it is intended to add this -Xdisableexcessivegc option regardless what variation we use, right?

Reply:
Correct. That's a requirement of those particular tests.